frontend/src/pages/post/Search/SearchPage.jsx

```diff
@@ -0,0 +1,26 @@
+import { useParams } from "react-router-dom"
+import postData from '../mock-data'
```

Comment:
James has refactored the code to use data from the DB rather than mock-data now, you'll need to incorporate that change into your branch before this can be merged I think